README.md

```diff
@@ -29,11 +30,11 @@ platform (authentication and authorization of the channel) and provide other com
 
 There is more information about specific topics in the following documents:
 
-* [User manual](doc/usermanual.md): library reference for IoTA implementation. Contains the complete list of library
```

Comment:
RTD adaptation should be done in a way it doesn't break markdown rendering. Using absolute references here would cause such break...

Is this necesary? Looking the paralell case in fiware-orion repo (that is already adapted to RTD but keeps the makrdown rendering fine) it wasn't needed and we are using relative links there.

Reply:
NTC - Yes this is necessary. The docs recommendations are in the process of being updated to prefer styled Readthedocs links from the README - the reason behind this is threefold.

* styled documentation is easier to read
* styled documentation appears to come from an official common source - e.g. using FIWARE fonts rather than Github fonts
* Readthedocs gives a Navigation sidebar and search functionality enabling users to find information more easily

Reply:
By the moment, let's keep both Markdown and RTD self-contained. This dual documentation approach has been working for years in Orion repository and I think it's the way to go. Both RTD and Markdown are good documentation systems and I prefer the user to choose which one she/he wants to use.

Thus, I'd suggest to rollbackt this change (and other in the PR that are replacing relative links in Markdwon with absolute links to RTD pages) and focus the PR in the mkdocs.yml, quote fixes and similar stuff. Thanks!

Reply:
Links Reverted - Fixed in #8befe9a -  IMHO, I  happen to disagree with this policy as I find CSS styled docs much easier to navigate, but until the guidelines have been updated I'm happy enough to revert the links for consistency reasons. I can always raise another PR